### DISCUSSION TITLE
How to update to the dev version of Optuna dashboard? Release of new version soon?

### DISCUSSION THREAD

**Discussion by simoneVU** [optuna/optuna-dashboard#139] [Q&A] [answered]
I was wondering how I can update my 0.4.2 version of Optuna-dashboard to the latest dev version of Optuna-dashboard found on GitHub. I still encounter the kind of annoying overlapping label for the hyperparameters importance plot.  I also wanted to know whether there is an exact date for the new release of Optuna-dashboard, looking forward for it!

Thanks in advance!

**Comment by c-bata** [comment, marked as answer]
We plan to release the new version 0.5.0 at the end of this month. You can install the latest dev version of optuna-dashboard like this:

```
$ git clone git@github.com:optuna/optuna-dashboard.git
$ cd optuna-dashboard
$ npm install
$ npm run build:prd
$ pip install .
```

**Reply by simoneVU** [reply]
Thanks for the clarification and the help! It worked!
